### PR TITLE
[Feat] Create EASE model

### LINKED ISSUE #4: [Feat] Create Ease model
## Background
- hidden layer 가 없으면서도, spare matrix 에 대해 강력한 성능을 자랑하는 EASE 모델 도입

## To Do
- [x] EASE paper review
- [ ] EASE dataset
- [ ] EASE dataloader 
- [x] EASE model 
- [ ] EASE trainer
- [x] EASE loss function type 1 : Frobenius norm 
- [ ] EASE loss function type 2 : Multinomial likelihood

### PR DESCRIPTION
**Pull request by AI-soyoung**
## Overview
- Closed Form 으로 작성된 EASE model 입니다.

## Change Log
- model/EASE.ipynb 가 추가됨

## To Reviewer
- 다른 파일과의 종속성 없이, .ipynb 만으로 동작합니다.

## Issue Tags
Close #4

### CODE REVIEW

**Comment by rlawldn11703** [comment]
Closed Form으로 뛰어난 성능을 내는 모델이라 신기하네요! 고생하셨습니다!!
